site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -445,6 +448,64 @@ For high availability deployments with purchased support:
 * The FusionAuth engineering team will consult on product implementation and can offer "best practices" advice to further ensure system stability.
 * We size the instance properly for expected traffic, in consultation with the customer.
 
+== Deployment IP Addresses
+
+You may need to know the IP addresses of the instances in your FusionAuth Cloud deployment. There are any number of reasons you might need this information, including:
+
+* Adding the IP addresses to the allow list of an email deliverability vendor.
+* Punching a hole in a corporate firewall to allow for webhook delivery.
+* Adding the IP address to a network access control list to enable access to an on-premises LDAP directory with a connector.
+
+[WARNING.warning]
+====
+The IP addresses of FusionAuth Cloud instances may change at any time. Prepare for this by scripting this procedure, running it regularly, and updating the allow list.
+
+Hostnames are immutable, and if you can use them instead of IP addresses, that will require less maintenance.
+====
+
+You can find the IP addresses for your deployment, but be aware that these addresses may change at any time.
+
+Use `dig` to find the address, based on your deployment hostname. Use the default hostname ending in `fusionauth.io`, not any custom domains you have added. Deployment nodes are named in a standardized manner.
```

Comment:
> Use the default hostname ending in `fusionauth.io`, not any custom domains you have added.

> Use the default hostname ending in `fusionauth.io` (as opposed to any custom domains you have added).

Reply:
`dig` is is a Linux or Unix command, and may not be available on all systems (such as Windows). If we are going to document how to resolve an IP address from a hostname, we should document an option for Windows, Linux and macOS. In practice Linux and macOS will likely be the same. We could also indicate they can use a DNS lookup tool or service.

Reply:
ok, used nslookup, which is cross platform.